src/lib/core/styles/tokens/field/_tokens.scss

```diff
@@ -0,0 +1,135 @@
+@use 'sass:map';
+@use '../../animation';
+@use '../../border';
+@use '../../elevation';
+@use '../../shape';
+@use '../../theme';
+@use '../../utils';
+
+$tokens: (
+  display: utils.module-val(field, display, block),
+  text-color: utils.module-val(field, text-color, theme.variable(text-high)),
+
+  // Spacing (label)
+  label-margin-inline: utils.module-val(field, label-margin-inline, 8px),
+  label-margin-block: utils.module-val(field, label-margin-block, 2px),
+  label-gap: utils.module-val(field, label-gap, 0.25em),
+
+  // Spacing (container)
+  container-padding: utils.module-val(field, container-padding, 12px),
+  container-padding-start: utils.module-ref(field, container-padding-start, container-padding),
+  container-padding-end: utils.module-ref(field, container-padding-end, container-padding),
+  container-gap: utils.module-val(field, container-gap, 8px),
+  start-padding-end: utils.module-ref(field, start-padding-end, container-gap),
+  end-padding-start: utils.module-ref(field, end-padding-start, container-gap),
+  end-padding-end: utils.module-ref(field, end-padding-end, container-gap),
+  popover-icon-padding-start: utils.module-ref(field, popover-icon-padding-start, container-gap),
+  popover-icon-padding-end: utils.module-ref(field, popover-icon-padding-end, container-gap),
+  accessory-padding: utils.module-ref(field, accessory-padding, container-gap),
+  accessory-padding-start: utils.module-ref(field, accessory-padding-start, accessory-padding),
+  accessory-padding-end: utils.module-ref(field, accessory-padding-end, accessory-padding),
+
+  // Spacing (support text)
+  support-text-padding-block: utils.module-val(field, support-text-padding-block, 2px),
+  support-text-gap: utils.module-val(field, support-text-gap, 8px),
+  support-text-padding-inline: utils.module-ref(field, support-text-padding-inline, container-padding),
+  support-text-padding-inline-start: utils.module-ref(field, support-text-padding-inline-start, support-text-padding-inline),
+  support-text-padding-inline-end: utils.module-ref(field, support-text-padding-inline-end, support-text-padding-inline),
+
+  // Spacing (pill)
+  pill-container-padding: utils.module-val(field, pill-container-padding, 16px),
+  pill-container-padding-start: utils.module-ref(field, pill-container-padding-start, pill-container-padding),
+  pill-container-padding-end: utils.module-ref(field, pill-container-padding-end, pill-container-padding),
+  
+  // Sizes
+  height-extra-small: utils.module-val(field, height-extra-small, 24px),
+  height-small: utils.module-val(field, height-small, 32px),
+  height-medium: utils.module-val(field, height-medium, 40px),
+  height-large: utils.module-val(field, height-large, 48px),
+  height-extra-large: utils.module-val(field, height-extra-large, 56px),
```

Comment:
Let's create some base-level tokens like `height` in this case to allow for adjusting a single token across all variants if desired, while still supporting individual variant-specific token overrides.

See other comment about needing to update our `override()` function implementation to support this, but our code should be structured this way to make it easier on consumers for overriding.